packages/manager/apps/web-ongoing-operations/src/pages/tracking/Tracking.tsx

```diff
@@ -0,0 +1,315 @@
+import { BaseLayout } from '@ovh-ux/manager-react-components';
+import React, { useEffect, useState } from 'react';
+import { useTranslation, getI18n } from 'react-i18next';
+import {
+  OdsButton,
+  OdsCard,
+  OdsDivider,
+  OdsIcon,
+  OdsMessage,
+  OdsSpinner,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import { useQuery } from '@tanstack/react-query';
+import { useNavigate, useParams } from 'react-router-dom';
+import {
+  ODS_CARD_COLOR,
+  ODS_TEXT_PRESET,
+  ODS_DIVIDER_COLOR,
+  ODS_MESSAGE_COLOR,
+} from '@ovhcloud/ods-components';
+import { formatDate } from '@/utils/utils';
+import { TrackingEnum } from '@/enum/tracking.enum';
+import { getOperationTrackingStatus } from '@/data/api/tracking';
+import Loading from '@/components/Loading/Loading';
+import SubHeader from '@/components/SubHeader/SubHeader';
+import { TOngoingOperations } from '@/types';
+import { getmeTaskDomainId } from '@/data/api/web-ongoing-operations';
+import { urls } from '@/routes/routes.constant';
+
+export default function Track() {
```

Comment:
Can you split this component implementation with multiple components to make it more readable ?